### DISCUSSION TITLE
2-Regression/1-Tools/notebook.ipynb is an empty file

### DISCUSSION THREAD

**Discussion by devDavide** [microsoft/ML-For-Beginners#659] [Q&A]
Hi there,at first thank you for this amazing course.
I was following the lectures and I saw that [2-Regression/1-Tools/notebook.ipynb](https://github.com/microsoft/ML-For-Beginners/blob/main/2-Regression/1-Tools/notebook.ipynb) is an empty file and it does not contain what explained in the documentation.
Could you please add the mentioned notebook to the folder ?

Many thanks

**Comment by my-run** [comment]
Follow the tutorials and code by yourself as a playground.
